.gitlab/container_build/docker_linux.yml

```diff
@@ -127,6 +127,21 @@ docker_build_agent7:
     TEST_IMG: "true"
     BUILD_ARG: --target release --build-arg PYTHON_VERSION=3 --build-arg DD_AGENT_ARTIFACT=datadog-agent_7*_amd64.deb
 
+single_machine_performance-push-main-sha:
+  stage: container_build
+  rules:
+    - !reference [.on_main_a7]
+  needs:
+    - docker_build_agent7
```

Comment:
Shouldn't the `needs` be for `single_machine_performance-amd64-a7`? After all, we need to wait for that one to finish to use it as baseline image in other pipelines IIUC.

Reply:
That's a good point. Updated.